### PR TITLE
More typings

### PR DESCRIPTION
**Pull request by miherlosev**
@LavrovArtem

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

File: src/client/sandbox/node/index.ts [hunk 1]
```diff
@@ -10,23 +10,38 @@ import * as domUtils from '../../utils/dom';
...
-    eventSandbox: any;
-    iframeSandbox: any;
-    shadowUI: any;
-    mutation: any;
     doc: any;
     win: any;
     element: any;
 
-    constructor (nodeMutation, iframeSandbox, eventSandbox, uploadSandbox, shadowUI, cookieSandbox) {
+    constructor (nodeMutation: NodeMutation,
```
Comment (LavrovArtem) [inline src/client/sandbox/node/index.ts]:
```ts
readonly nodeMutation: NodeMutation
```

File: src/client/sandbox/node/index.ts [hunk 2]
```diff
@@ -10,23 +10,38 @@ import * as domUtils from '../../utils/dom';
...
+import UploadSandbox from '../upload';
+import ShadowUI from '../shadow-ui';
+import CookieSandbox from '../cookie';
+/*eslint-enable no-unused-vars*/
 
 const ATTRIBUTE_SELECTOR_REG_EX          = /\[([\w-]+)(\^?=.+?)]/g;
 const ATTRIBUTE_OPERATOR_WITH_HASH_VALUE = /^\W+\s*#/;
 
 export default class NodeSandbox extends SandboxBase {
+    mutation: NodeMutation;
```
Comment (LavrovArtem) [inline src/client/sandbox/node/index.ts]: remove

File: src/client/sandbox/node/index.ts [hunk 3]
```diff
@@ -10,23 +10,38 @@ import * as domUtils from '../../utils/dom';
...
-    constructor (nodeMutation, iframeSandbox, eventSandbox, uploadSandbox, shadowUI, cookieSandbox) {
+    constructor (nodeMutation: NodeMutation,
+                 iframeSandbox: IframeSandbox,
+                 private readonly _eventSandbox: EventSandbox, //eslint-disable-line no-unused-vars
+                 private readonly _uploadSandbox: UploadSandbox, //eslint-disable-line no-unused-vars
+                 shadowUI: ShadowUI,
+                 private readonly _cookieSandbox: CookieSandbox) { //eslint-disable-line no-unused-vars
         super();
 
+        this.mutation              = nodeMutation;
```
Comment (LavrovArtem) [inline src/client/sandbox/node/index.ts]: remove

File: src/client/sandbox/node/live-node-list/html-collection-wrapper.ts [hunk 4]
```diff
@@ -8,7 +8,7 @@ const ELEMENTS_WITH_NAME_ATTRIBUTE     = ['button', 'fieldset', 'form', 'iframe'
 const COLLECTION_PROTO_GETTERS_RESERVE = 10;
 let collectionProtoGettersCount        = 0;
 
-export default function HTMLCollectionWrapper (collection, tagName) {
+export default function HTMLCollectionWrapper (collection, tagName: string) {
```
Comment (LavrovArtem) [inline src/client/sandbox/node/live-node-list/html-collection-wrapper.ts]: collection: HTMLCollection

File: src/client/sandbox/shadow-ui.ts [hunk 5]
```diff
@@ -106,14 +107,14 @@ export default class ShadowUI extends SandboxBase {
...
-        this.bodyCreatedEventCallback = ({ body }) => this.markShadowUIContainers(this.document.head, body);
+        this.bodyCreatedEventCallback = body => this.markShadowUIContainers(this.document.head, body);
     }
 
     static _filterElement (el) {
         return el && domUtils.isShadowUIElement(el) ? null : el;
     }
 
-    _filterList (list, listLength, predicate) {
+    _filterList (list, listLength: number, predicate) {
```
Comment (LavrovArtem) [inline src/client/sandbox/shadow-ui.ts]: Add the `private` keyword before all functions that start with _

File: src/client/sandbox/node/live-node-list/dom-mutation-tracker.ts [hunk 6]
```diff
@@ -16,12 +16,12 @@ class DOMMutationTracker {
...
     }
 
-    _updateVersion (tagName) {
+    _updateVersion (tagName: string) {
         if (tagName in this._mutations)
             this._mutations[tagName].increment();
     }
 
-    _processElement (el) {
+    _processElement (el: any) {
```
Comment (LavrovArtem) [inline src/client/sandbox/node/live-node-list/dom-mutation-tracker.ts]: The `any` type is default type. Why do we need to specify it?

Comment (miherlosev) [reply src/client/sandbox/node/live-node-list/dom-mutation-tracker.ts]: image

**Pull request merged**